frontend/components/DocumentEditToolbar/DocumentEditToolbar.jsx

```diff
@@ -111,7 +117,7 @@ export default class DocumentListToolbar extends Component {
         <div class={styles['padded-group']}>
           <ButtonWithOptions
             accent="save"
-            disabled={disabled}
+            disabled={hasValidationErrors || disabled}
```

Comment:
Also, deciding whether `hasConnectionIssues` is used needs to be clear. It should be reusable and therefore explicit